bin/sentieon-genomics-202010.02/lib/python/sentieon/vcflib/tribble.py

```
@@ -0,0 +1,320 @@
+        self.longest = 0
+        self.count = 0
+        self.blocks = []
+        self.density = density
+
+    def decode(self, data, off):
+        chrom,_ = data[off:].split(b'\0',1)
+        off += len(chrom) + 1
+        self.chrom = chrom.decode()
+        s = struct.Struct('<iiiii')
+        self.width, bins, self.longest, _, self.count = s.unpack_from(data, off)
+        off += s.size;
```

Ruff (0.8.2)

Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)